### DISCUSSION TITLE
Install Error

### DISCUSSION THREAD

**Discussion by danielschramm** [posixx/hass_addons#1] [Q&A]
Hello,

I tried to add the Plugin to Homeassistant and got the following error:
The command '/bin/ash -o pipefail -c pip3 install apscheduler' returned a non-zero code: 1

Could it be a version conflict or free disk size?

Thanks,

Daniel

Version	core-2021.12.10
Installation Type	Home Assistant OS
CPU Architecture	x86_64
Host Operating System	Home Assistant OS 7.2
Update Channel	stable
Supervisor Version	supervisor-2021.12.2
Docker Version	20.10.9
Disk Total  	30.8 GB
Disk Used	21.6 GB

**Comment by posixx** [comment]
Hello Daniel,

I really have no idea.. i didn't update the addons for a while because they simply work.. Which plugin does give you this error?

Frans

**Comment by danielschramm** [comment]
Hello,
it's the addon alpha2mqtt which causes this error.
Thanks,

Daniel
